app/src/main/res/layout/mozac_browser_menu_item_switch.xml

```diff
@@ -2,7 +2,7 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
   -  License, v. 2.0. If a copy of the MPL was not distributed with this
   -  file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<Switch xmlns:android="http://schemas.android.com/apk/res/android"
+<com.google.android.material.switchmaterial.SwitchMaterial xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Do we need the material one? Why not SwitchCompat?

Reply:
I checked the base theme of fenix and it uses this `Theme.MaterialComponents.DayNight.NoActionBar.Bridge`
Also for difference between `SwitchMaterial and SwitchCompat` , SwitchMaterial is just wrapper over SwitchCompat
https://stackoverflow.com/questions/63191570/warning-use-switchcompat-from-appcompat-or-switchmaterial-from-material-librar

Reply:
> SwitchMaterial is just wrapper over SwitchCompat

In general, the extra wrapper stuff over can cause theming problems if the material one does more than was intended from the original implementation here.

In this case, you don't need to worry about it since the [`Mozac.Browser.Menu.Item.Text`](https://searchfox.org/mozilla-mobile/source/android-components/components/browser/menu/src/main/res/values/style.xml#34) style here extends from `TextAppearance.Material.Menu` so it should be fine.

Reply:
I may be missing something, but its not `Text` its a `Switch`

Reply:
On line 6, the styling used is for `Text`.

Reply:
Umm I did not change to `SwitchMaterial` for text styling. There was one issue for this in `lint-baseline.xml`

![image](https://user-images.githubusercontent.com/24780524/133299396-6295c2a6-05ae-41a6-97b3-7aa6c58cd8dd.png)

Reply:
On line 6, the styling we use can conflict with the lint changes you made based on the internals of the `SwitchMaterial`.

We can talk about it in more detail on Matrix if you're interested, but I think you're fine here. :)